packages/api/test/utils/checkAgainstSpec.ts

```diff
@@ -19,17 +19,35 @@ ajv.addKeyword({
 
 ajv.addFormat("hex", /^0x[a-fA-F0-9]+$/);
 
+/**
+ * A property that will be filtered during tests execution.
```

Comment:
Could more context here on why this is useful / needed (what you already wrote in the PR description)

Reply:
Done